src/main/kotlin/org/rust/ide/colors/RustColors.kt

```diff
@@ -9,11 +9,11 @@ import com.intellij.openapi.editor.DefaultLanguageHighlighterColors as Default
  */
 enum class RustColor(humanName: String, val default: TextAttributesKey) {
     IDENTIFIER            ("Identifier",                  Default.IDENTIFIER),
-    FUNCTION_DECLARATION  ("Function declaration",        Default.FUNCTION_DECLARATION),
-    INSTANCE_METHOD       ("Instance method declaration", Default.INSTANCE_METHOD),
-    STATIC_METHOD         ("Static method declaration",   Default.STATIC_METHOD),
-    PARAMETER             ("Function parameter",          Default.PARAMETER),
-    SELF_PARAMETER        ("Self parameter",              Default.KEYWORD),
+    FUNCTION_DECLARATION  ("Function",                    Default.FUNCTION_DECLARATION),
+    INSTANCE_METHOD       ("Method",                      Default.INSTANCE_METHOD),
+    STATIC_METHOD         ("Associated function",         Default.STATIC_METHOD),
```

Comment:
>I think we should use "associated" instead of "static" everywhere in the code.

It's just a general comment and a note to myself, there's no need to fix everything in this PR